File: fabric/src/client/java/juuxel/adorn/client/AdornClientInit.java
package juuxel.adorn.client;

import juuxel.adorn.client.book.BookManager;
import juuxel.adorn.client.gui.screen.AdornMenuScreens;
import juuxel.adorn.client.renderer.AdornRenderers;
import juuxel.adorn.client.resources.AdornClientResources;
import net.fabricmc.api.ClientModInitializer;

public final class AdornClientInit implements ClientModInitializer {
    @Override
    public void onInitializeClient() {
        AdornRenderers.init();
        AdornMenuScreens.register();
        AdornClientNetworking.init();
        AdornClientResources.init();
        ClientEvents.init();
        AdornModels.init();
        BookManager.setupTooltipProvider();
    }
}

File: fabric/src/client/kotlin/juuxel/adorn/client/AdornClientInit.kt
package juuxel.adorn.client

import juuxel.adorn.client.gui.screen.AdornMenuScreens
import juuxel.adorn.client.lib.AdornBlocksClient
import juuxel.adorn.client.lib.AdornEntitiesClient
import juuxel.adorn.client.lib.AdornResources
import juuxel.adorn.client.lib.ClientNetworking
import net.fabricmc.api.ClientModInitializer

object AdornClientInit : ClientModInitializer {
    override fun onInitializeClient() {
        AdornBlocksClient.init()
        AdornEntitiesClient.init()
        AdornMenuScreens.register()
        ClientNetworking.init()
        AdornResources.init()
        ClientCompatInit.init()
    }
}
